jobs/build/update-base-images/build.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+set -e
+set -o pipefail
```

Comment:
We should standardize on

`set -euxo pipefail`

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/